### DISCUSSION TITLE
The component update data in IFRAM is out of sync

### DISCUSSION THREAD

**Discussion by declanchiu** [pmndrs/zustand#1636] [Q&A]
I have two components that call the set method of zustand create inside an iframe. However, the values obtained in the normal component are still unchanged. Is there a good solution for this kind of zustand?

**Comment by dai-shi** [comment]
You need to make communication between frames. Maybe with postMessage or BroadcastChannel. I too wonder if someone does something like that.

**Comment by brunoalano** [comment]
Did you found any solution @declanchiu?
